app/serializers/activitypub/accept_follow_serializer.rb

```diff
@@ -3,7 +3,8 @@
 class ActivityPub::AcceptFollowSerializer < ActiveModel::Serializer
   attributes :id, :type, :actor
 
-  has_one :object, serializer: ActivityPub::FollowSerializer
+  attribute :remote_id, key: :object
```

Comment:
It makes any `Follow` request made before this change fail, right? I do not think it is OK. We need a fallback.